swarm-event-schema.json

```diff
@@ -197,6 +197,9 @@
         "trackId" : {
            "type": "number" 
         },
+        "numberPlate": {
```

Comment:
Had to look this up, we stay with British English https://en.wikipedia.org/wiki/Vehicle_registration_plate
😄